tpu-client-next/src/logging.rs

```diff
@@ -4,17 +4,11 @@
 //! to use either the `log` crate (default) or the `tracing` crate.
 //! The features are mutually exclusive - only one can be enabled at a time.
 
-#[cfg(feature = "log")]
+#[cfg(not(feature = "tracing"))]
 pub use log::{debug, error, info, trace, warn};
 #[cfg(feature = "tracing")]
 pub use tracing::{debug, error, info, trace, warn};
 
-#[cfg(not(any(feature = "log", feature = "tracing")))]
-compile_error!("Either 'log' or 'tracing' feature must be enabled");
-
-#[cfg(all(feature = "log", feature = "tracing"))]
-compile_error!("'log' and 'tracing' features are mutually exclusive");
-
```

Comment:
Two more minor changes for this file:
1. I think we should adjust the comment at the top since `log` is no longer a feature
```rust
//! Logging abstraction module that supports both `log` and `tracing` libraries.
//!
//! This module provides a unified logging interface. The `log` crate is used by
//! default, but the `tracing` crate can be used instead by enabling the
//! `tracing` feature for this crate.
```
2. I'm inclined to think we can rip the unit test in this file, but I guess no harm if we leave it. I guess curious what you two (Chido & Kirill) think